src/main/java/view/OutputView.java

```diff
@@ -0,0 +1,65 @@
+package view;
+
+import domain.BlockType;
+import domain.Line;
+import domain.Players;
+
+import java.util.List;
+
+public class OutputView {
+    public static final String FIRST_FORMAT = "%-5s";
+    public static final String MIDDLE_FORMAT = "%6s";
+    public static final String LAST_FORMAT = "%5s";
+    public static final String VERTICAL_BAR = "|";
+
+    private static void middlePlayersPrint(List<String> names, int index) {
+        if (isMiddlePlayer(names, index)) {
+            System.out.print(String.format(MIDDLE_FORMAT, names.get(index)));
```

Comment:
IDE가 무의미한 호출이라고 안내해주네요 :) 같은 의미를 가진 `printf()`가 있기 때문입니다
```suggestion
            System.out.printf(MIDDLE_FORMAT, names.get(index));
```

Reply:
넵!! 수정하겠습니다!